slackbridge/messages.py

```diff
@@ -165,6 +191,13 @@ def _post_to_irc(self, channel_name, user_bot):
             self.raw_message['text'],
         )
 
+    def _post_pm_to_irc(self, channel_name, user_bot):
+        user_bot.post_to_irc(
+            user_bot.msg,
+            channel_name,
```

Comment:
For a PM, this is actually the name of the user to send it to instead of a channel name, right? I think naming it something like `irc_recipient` or something would make this clearer.